### PR TITLE
CA 0.6 FAQ update

### PR DESCRIPTION
**Pull request by MaciekPytel**
Added questions about new features coming in 0.6: metrics / livenessProbe and balancing node groups.

### CODE REVIEW

File: cluster-autoscaler/FAQ.md [hunk 1]
```diff
@@ -94,6 +104,39 @@ They are also not particularly suited to use with Kubernetes in general.
 
 ****************
 
+# How to?
```
Comment (MaciekPytel) [inline cluster-autoscaler/FAQ.md]: I feel those questions don't fit into any existing section, but I'm not too happy with the name of this one. I'm happy to rename it if you have any better ideas. 

@mwielgus @aleksandra-malinowska

File: cluster-autoscaler/FAQ.md [hunk 2]
```diff
@@ -94,6 +104,39 @@ They are also not particularly suited to use with Kubernetes in general.
...
+### I'm running cluster with nodes in multiple zones for HA purposes. Is that supported by Cluster Autoscaler?
+CA 0.6 introduced `--balance-similar-node-groups` flag to support this use-case. If you set the flag to true
+CA will automatically identify node groups using the same instance types and
+having the same set of labels (except for automatically added zone labels) and try to
+keep the size of those node groups balanced.
+
+This does not guarantee similar node groups will have exactly the same sizes:
+* Currently the balancing is only done at scale-up. Cluster Autoscaler will
+  still scale-down underutilized nodes regardless of relative size of underlying
+  node groups. We plan to take balancing into account in scale-down in future.
```
Comment (mwielgus) [inline cluster-autoscaler/FAQ.md]: the future

File: cluster-autoscaler/FAQ.md [hunk 3]
```diff
@@ -94,6 +104,39 @@ They are also not particularly suited to use with Kubernetes in general.
...
+* Cluster Autoscaler will only add as many nodes as required to run all existing
+  pods. If the number of nodes is not divisible by number of balanced node
+  groups some groups will get 1 more node than others.
+* Cluster Autoscaler will only balance between groups that can support all
+  pending pods. If you run pods that can only go to a single node group (for example
```
Comment (mwielgus) [inline cluster-autoscaler/FAQ.md]: the same set of pending pods.

File: cluster-autoscaler/FAQ.md (hunk 2 above)
Comment (MaciekPytel) [inline cluster-autoscaler/FAQ.md]: done

File: cluster-autoscaler/FAQ.md (hunk 3 above)
Comment (MaciekPytel) [inline cluster-autoscaler/FAQ.md]: done

**Pull request merged**